### ISSUE TITLE
App not “remembering” where you were

### ISSUE THREAD

**Issue by ben-zacharia** [christianselig/apollo-bugs#628]
**Describe the bug**
Long time user!
Not quite sure how to put this, I haven’t done mobile dev but when you switch between apps on your phone typically the recently used apps are kept in memory or something so that when you navigate back to them you are put back where you were. This used to work well with Apollo, but with the last update or two this hasn’t worked. You navigate away and come back to the app and immediately the cache is gone and your back on the home screen. 

**To Reproduce**
-open the app
-navigate anywhere, a subreddit, some comments
-click the home button and open another app (messages or whatever)
-double tap the home button and go back to Apollo
-you’re back on the home page not where you were

**iOS version**: Version of iOS this bug occurred on (can be found in Settings.app > General)
- 13.3
**Apollo version**: Version of Apollo this bug occurred on (can be found in Apollo's Settings tab > About)
- submitting on mobile but it’s the current version as of 2/14/20
**iOS device**: For example iPhone XS, iPad mini, etc. 
-iPhone8
**Jailbroken**: Whether or not your device is jailbroken
Nope
**Reproducible:** Can you reproduce it every time? Or is it random?
Seems to happen every time, pretty sure it’s a regression as it definitely used to work. 

**Expected behavior**
I expect that, within reason, Apollo will put me back where I was. Typically the last few apps (maybe 3-4) are ready to go but it depends on the app. 

**Screenshots**
If possible, add screenshots to help show the problem.
Not sure this would help here

**Video**
If possible, attach a video to help show the problem. You can attach a video by uploading it to Streamable, Gfycat, YouTube, etc. and pasting the URL here.

**Additional context**
Hope that helps! You can message back here and I’ll try and remember to check in. I don’t come to GitHub much

**Comment by ben-zacharia** [comment]
Of course, now that I submit this it doesn’t seem to happen immediately but I definitely noticed that it was a lot quicker than usual that the app forgot where you were

**Comment by kidGodzilla** [comment]
I've also have this behavior for about 1 month.

When you switch between apps, it is now very likely that I see a loading screen and get sent back to the top of Home.

iOS 13.3, iPhone 7+, Paid subscription.

Basically, every time I lock my phone or go to the task switcher to switch to another app, the app state gets reset. Usually notice it when I'm deep in a comment thread and I lock my phone to get off the train, etc. -- Unlocking the phone re-launches the app in a fresh state.

I believe this first occurred after upgrading to iOS 13.3 (I updated in January). It may also be occurring when available memory is low on the device (it seems to happen less frequently if I make sure only 2-3 apps are open).

**Comment by ghost** [comment]
Hi,

I have a screen recording of this behavior. Is there anything else that I can provide that will help?

Specs:
- iPhone model: 7
- iOS version: 13.3.1
- Jailbreak: Nah
- App version: 1.7.3
- App subscription: Apollo Pro

Reproduce:
What op wrote. See [this](https://imgur.com/a/qHW1eL2) video.

**Comment by mikerastiello** [comment]
I am also seeing this issue on the iPad only. 

I notice it when:
- Browsing any subreddit, /r/all, doesn't matter
- Leave app to go to another app OR shut device screen off
- Go back to Apollo and my place has (behind the scenes) scrolled up a number of posts

My device:
- iPad 11 pro
- iPadOS 13.3.1
- Apollo v1.7.3

My iPhone (11 Pro Max / iOS 13.3.1 / Apollo v1.7.3) runs as expected.

**Comment by devanshuDesai** [comment]
Been facing exact same issue for at least 3 months now

- iPhone 7+
- iOS 13.4
- Apollo 1.7.3

**Comment by toddyco** [comment]
This is still occurring with latest version on iPad on latest version of iPad OS. Leaving the app for even a split second and returning will cause the scroll position in a subreddit (including r/all) to change to a position further up from where you were. The further you had scrolled in the subreddit, the greater the offset. Thus I’m sure there’s a bad math calculation happening somewhere to determine how far you had scrolled.

**Comment by cht87** [comment]
Problem still persists. 

Iphone 7+
Latest iOS and app version. 

Paid user. 

If this persists I will have to refund.
